addon/components/polaris-empty-state/details.js

```diff
@@ -1,46 +1,7 @@
 import Component from '@ember/component';
 import { tagName, layout as templateLayout } from '@ember-decorators/component';
 import layout from '../../templates/components/polaris-empty-state/details';
-import TaglessCssDeprecation from '../../mixins/tagless-css-deprecation';
 
 @tagName('')
 @templateLayout(layout)
-export default class PolarisEmptyStateDetails extends Component.extend(
-  TaglessCssDeprecation
-) {
-  /**
-   * The empty state heading
-   *
-   * @type {String}
-   * @default null
-   */
-  heading = null;
-
-  /**
-   * Elements to display inside empty state
-   *
-   * This component can be used in block form,
-   * in which case the block content will be used
-   * instead of `text`
-   *
-   * @type {String}
-   * @default null
-   */
-  text = null;
-
-  /**
-   * Primary action for empty state
-   *
-   * @type {Object}
-   * @default null
-   */
-  action = null;
-
-  /**
-   * Secondary action for empty state
-   *
-   * @type {Object}
-   * @default null
-   */
-  secondaryAction = null;
-}
+export default class PolarisEmptyStateDetails extends Component {}
```

Comment:
Do we still need this JS file? Also curious why the deprecation warning's been removed 🤔

Reply:
Seems still needed in addons?!
This is just internal, so no need to repeat the deprecations and documenting properties since we do in the parent component. Ideally, this would be just a tagless component